CONTRIBUTING.md

```diff
@@ -39,10 +39,35 @@ contributors a chance to resolve the vulnerability and issue a new release prior
 to any public exposure; this helps protect Apigility users, and provides them
 with a chance to upgrade and/or update in order to protect their applications.
 
-For sensitive email communications, please use 
+For sensitive email communications, please use
 [our PGP key](http://framework.zend.com/zf-security-pgp-key.asc).
 
-## RUNNING TESTS
+## DOCKER FOR DEVELOPMENT
+
+This library requires a running instance of Mongo in order to run and pass
+the unit tests.  It is not expected for each developer to configure their
+individual machine to match this environment so Docker is provided.
+
+### RUNNING DOCKER-COMPOSE
+
+You will need docker-compose installed on your machine.
+Docker supports many PHP versions.  You may specify the version of PHP you
+want (default:7.2) to build against when running `docker-compose` and flag XDEBUG to
+install (1) or not (default:0).
+
+Next, from the root directory of this project, run
+
+```
+docker-compose build --build-arg PHP_VERSION=7.2 --build-arg XDEBUG=0 --no-cache php
```

Comment:
I think we can simplify it here:
```console
$ docker-compose build
```

because as said above PHP_VERSION=7.2 and XDEBUG=0 are default values. Then we should say if you want customize your container with different php version or enabled xdebug run:
```console
$ docker-compose build --build-arg PHP_VERSION=7.1 --build-arg XDEBUG=1
```

(not sure also if we need `--no-cache` param - I need to check it).